CorePluginFramework/src/au/gov/asd/tac/constellation/plugins/gui/LocalDateInputPane.java

```diff
@@ -110,5 +114,14 @@ public LocalDate fromString(final String s) {
         });
 
         getChildren().add(field);
+        parameterId = parameter.getId();
+        List<String> dateTimeRecentValues = RecentParameterValues.getRecentValues(parameterId);
+        if (dateTimeRecentValues != null) {
+            if (dateTimeRecentValues.size() > 1) {
+                parameter.setStringValue(dateTimeRecentValues.get(1));
+            } else {
+                parameter.setStringValue(dateTimeRecentValues.get(0));
+            }
```

Comment:
if-else can be converted to tertiary (similar to suggestion for `ColorInputPane`)